apps/cowswap-frontend/src/modules/orders/index.ts

```diff
@@ -16,3 +16,4 @@ export { emitFulfilledOrderEvent } from './utils/emitFulfilledOrderEvent'
 export { emitCancelledOrderEvent } from './utils/emitCancelledOrderEvent'
 export { emitExpiredOrderEvent } from './utils/emitExpiredOrderEvent'
 export { emitPresignedOrderEvent } from './utils/emitPresignedOrderEvent'
+export { emitBridgingSuccessEvent } from './utils/emitBridgingSuccessEvent'
```

Comment:
nit: could sort alphabetically and place `emitBridgingSuccessEvent` first

Reply:
Fixed, thanks!